beetsplug/bpd/__init__.py

```diff
@@ -640,8 +640,8 @@ class Command(object):
     """A command issued by the client for processing by the server.
     """
 
-    command_re = re.compile(br'^([^ \t]+)[ \t]*')
-    arg_re = re.compile(br'"((?:\\"|[^"])+)"|([^ \t"]+)')
+    command_re = re.compile(r'^([^ \t]+)[ \t]*')
+    arg_re = re.compile(r'"((?:\\"|[^"])+)"|([^ \t"]+)')
```

Comment:
I think you're right. I'll give it a go. There were so many little pieces just ot make it work, that I didn't get a chance to go over fix to see why things are the way they are.